tests/e2e/sideNav-dropdown-test.js

```diff
@@ -0,0 +1,31 @@
+import createServer from '../createServer'
+import { Selector, ClientFunction } from 'testcafe'
+require('dotenv').config()
+
+const getWindowLocation = ClientFunction(() => window.location)
+
+const hostname = `localhost`
+const port = 3002
+const baseUrl = `${hostname}:${port}`
+
+let server
+
+fixture`Side Nav Menu`.page`${baseUrl}`
+  .before(async t => {
+    server = createServer(port)
+  })
+  .after(() => server.close())
+
+test('we are on the homepage', async t => {
+  const location = await getWindowLocation()
+  await t.expect(location.href).contains(baseUrl)
+})
+
+test('a hamburger is present and opens the side nav', async t => {
+  const hamburger = await Selector('[class^="Hamburger"').filterVisible()
+  await t.expect(hamburger.exists).ok()
+  await t.click(hamburger)
+
+  const sideNav = await Selector('[class^="SideNav"]').filterVisible()
+  await t.expect(sideNav.exists).ok()
+})
```

Comment:
So the main reason I wanted to get some tests for the sidenav was so we could catch any functionality errors. 

Lets add another one

`a dropdown opens when the dropdown title is clicked`

Reply:
We have plenty of tests now:
 ✓ we are on the homepage
 ✓ a hamburger is present on the page
 ✓ the side nav is not expanded at its initial state
 ✓ when the hamburger button is clicked the side nav expands
 ✓ clicking a side Nav link redirects to the correct url and closes the side Nav
 ✓ when the side nav expands for the first time no sub-item is shown by default
 ✓ when an item is clicked its sub-items appear
 ✓ clicking a sub-item redirects to the correct url and closes the side Nav
 ✓ After a sub-item has been clicked and if the sideNav is re-opened, the subitems are still visible and the selected one highlighted

PS: We looked at testcafe-react-selectors but in the end, it was easier and more convenient to use the API made available by Testcafe.